draft-ietf-tls-esni.md

```diff
@@ -998,6 +998,9 @@ If the client-facing server rejected ECH, or if the first ClientHello did not
 include an "encrypted_client_hello" extension, the client-facing server
 proceeds with the connection as usual. The server does not decrypt the
 second ClientHello's ECHClientHello.payload value, if there is one.
+If the client-facing server rejected ECH and the first ClientHello included an
+"encrypted_client_hello" extension, the client-facing server MAY include
+the "encrypted_client_hello" extension with a payload of 8 random bytes.
```

Comment:
* In which message may the extension appear? ServerHello, HelloRetryRequest, or both?
* I think we should make it clear that the purpose is to grease. Maybe point to Security Considerations and expand the relevant section to talk about this?

Reply:
Good suggestions -- I moved some stuff around to try and clarify.